src/services/microservice-service.ts

```diff
@@ -55,7 +55,8 @@ export class MicroserviceService {
 
     this.addMicroserviceDescriptor({
       ...ms,
-      healthCheckPath: ms.healthCheckPath ?? DEFAULT_HEALTH_CHECK_PATH
+      healthCheckPath: ms.healthCheckPath ?? DEFAULT_HEALTH_CHECK_PATH,
+      healthCheckIngress: ms.healthCheckIngress ?? HealthCheckIngress.Canonical
```

Comment:
ok for input but in output when you run `ent bundle pack` you should find your property in output (so`.entando/output/plugins/plugin-desc.yaml`) in that way the CM can processing the CR when it is applied.
try and let me know

Reply:
let's discuss that later. Not sure to get it

Reply:
done